package.json

```diff
@@ -37,7 +38,7 @@
     "ts-loader": "^8.2.0",
     "typescript": "^4.3.5",
     "webpack": "^5.50.0",
-    "webpack-cli": "^3.3.12",
+    "webpack-cli": "^4.10.0",
```

Comment:
The `webpack-dev-server` has been updated in dependabot, and `webpack-cli` needs to be updated as well.